### ISSUE TITLE
Method send4Direct always return false (don't work)

### ISSUE THREAD

**Issue by ant-kor** [afrad/angular2-websocket#54]
Issue in `if (self.socket.readyState !== self.readyStateConstants.RECONNECT_ABORTED) {
            return false;
        } else {
            self.fireQueue();
            return true;
        }`

When the connection is open:
    - self.socket.readyState is 1
    - self.readyStateConstants.RECONNECT_ABORTED is 4

self.fireQueue(); - never run

I think it should be like `if (self.socket.readyState === self.readyStateConstants.RECONNECT_ABORTED)`

**Comment by stestaub** [comment]
I agree with that. I think an even better souloution could be to check for OPEN:

```
f (self.socket.readyState == self.readyStateConstants.OPEN) {
    self.fireQueue();
    return true;
} else {
  return false;
}
```

I could provide a pull request If you agree with this fix

**Comment by ant-kor** [comment]
Please make a pull request.

**Comment by raugaral** [comment]
Pull request
https://github.com/afrad/angular2-websocket/pull/72
